data/mil2525d/utilities/style-utilities/merge-stylx-utilities/Readme.md

```diff
@@ -3,38 +3,53 @@
 ## Purpose
 
 * Merges:
-* A version of the .stylx that is edited and maintain in ArcGIS Pro
+    * A version of the .stylx that is edited and maintain in ArcGIS Pro
     * See: [mil2525d-lines-areas-labels-base-template.stylx](https://github.com/ArcGIS/military-features-pro-data/blob/master/data/core_data/stylxfiles/mil2525d-lines-areas-labels-base-template.stylx)
 * -with-
-* A version that is automatically generated from [marker symbols and meta-data-tagged source data](https://github.com/Esri/joint-military-symbology-xml)
-    * For more information on this process/version, see :
+    * A version of the .stylx file with the SVG/EMF marker/points-only icons. This version is normally automatically generated from SVG icon marker symbols and meta-data-tagged source data from the [joint-military-symbology-xml repository](https://github.com/Esri/joint-military-symbology-xml)
+    * For more information on this data/process/version, see :
         * [joint-military-symbology-xml repo](https://github.com/Esri/joint-military-symbology-xml)
         * [style-file-utilities](../style-file-utilities)
 
 ## Requirements
 
-* 2 Versions of the .stylx files:
-    * A Pro/Runtime stylx file with all of the 2525D Point Icons called `mil2525d-points-only.dat`
-        * This would have been created [by following the steps/readme here](https://github.com/ArcGIS/military-features-pro-data/tree/master/source/utilities)
-    *  A version of the .stylx that is edited and maintained in ArcGIS Pro: [mil2525d-lines-areas-labels-base-template.stylx](https://github.com/ArcGIS/military-features-pro-data/blob/master/data/core_data/stylxfiles/mil2525d-lines-areas-labels-base-template.stylx)
+* `(1)` 2 Versions of the .stylx files:
+    * A version of the .stylx file with the SVG/EMF icons 
+        * A Pro/Runtime stylx file with all of the 2525D Point Icons called `mil2525d-points-only.stylx`
+        * This would have been created [by following the steps/readme here](../../utilities)
+        * If you don't already have this, you can create it by importing the .style file with this icons, ["Military 2525Delta All.style"](../../../core_data/stylefiles/Military%202525Delta%20All.style), into ArcGIS Pro and renaming the resulting file to `mil2525d-points-only.stylx` 
```

Comment:
Unclear Typo: '.style file' isn't it '.stylx file?' In addition, 'with this icons' doesn't initially make sense. Could it be changed to 'with the icons' or 'with these icons' ?